app/views/movements/_form.html.erb

```diff
@@ -0,0 +1,16 @@
+<%= form_for @movement do |f|  %>
+  <% if @movement.errors.any? %>
```

Comment:
What do you think about moving this logic to a helper similar as the ' error_tag'?